Formula/op25.rb

```diff
@@ -0,0 +1,65 @@
+class Op25 < Formula
+  desc "Software-defined analyzer for APCO P25 signals"
+  homepage "https://osmocom.org/projects/op25/wiki"
+  url "git://op25.osmocom.org/op25.git"
+
+  depends_on "cmake" => :build
+  depends_on "cppunit" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "gnuradio"
+  depends_on "gr-osmosdr"
+  depends_on "itpp"
+
+  def install
+    # TODO: apply patch if gnuradio >= 3.8?
+
+    args = std_cmake_args + %w[
+      -Wno-dev
+    ]
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+
+    # Configure PYTHONPATH for runtime dependencies
+    [
+      Formula["gnuradio"].lib,
+      Formula["gr-osmosdr"].lib,
+      lib,
+    ].each do |dep|
+      Pathname.glob(dep/"python*/site-packages").each do |path|
```

Comment:
Would using `Language::Python.major_minor_version` similar to https://github.com/Homebrew/homebrew-core/blob/master/Formula/urh.rb#L32 be a better option here?